.ci_support/osx64.yaml

```diff
@@ -8,3 +8,9 @@ channel_sources:
   - conda-forge
 target_platform:
   - osx-64
+build_platform:
+- osx-arm64
+
+CONDA_BUILD_SYSROOT:
+- /opt/MacOSX10.9.sdk
+
```

Comment:
We don't allow for changes in the CI template at the moment. The "right" way to byukd osx-arm64 for now is to get the feedstock in first, then add the package to this list:  https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/migrations/osx_arm64.txt

Reply:
so just get rid of lines 11/12?

Reply:
All the lines you added here must be removed. We do not allow for CI config changes. If you need another SDK please take a look at: https://conda-forge.org/docs/maintainer/knowledge_base.html#requiring-newer-macos-sdks